worldengine/simulations/erosion.py

```diff
@@ -152,9 +152,7 @@ def river_sources(world, water_flow, water_path):
                 while not neighbour_seed_found:
 
                     # have we found a seed?
-                    if world.is_mountain((cx, cy)) and \
-                            water_flow[cy, cx] >= RIVER_TH:
-
+                    if world.is_low_mountain((cx, cy), atleast=True) and water_flow[cy, cx] >= RIVER_TH:
```

Comment:
I decided that rivers shouldn't start on hills anymore. Higher mountains would be an option as well.